spec/opportunities/user_views_all_opportunities_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature "user views upcoming opportunities on site" do
+
+  let!(:user) { FactoryGirl.create(:user, volunteer: false) }
+  let!(:organization) { FactoryGirl.create(:organization) }
+  let!(:org_admin) { FactoryGirl.create(
+    :org_admin,
+    user: user,
+    organization: organization)
+    }
```

Comment:
`}` at 11, 4 is not aligned with `let!(:org_admin) { FactoryGirl.create(` at 7, 2.